sbin/dhclient/conflex.c

```diff
@@ -467,6 +467,8 @@ intern(char *atom, int dfv)
 			return (REBOOT);
 		if (!strcasecmp(atom + 1, "eject"))
 			return (REJECT);
+		if (!strcasecmp(atom + 1, "fc3118-auth"))
```

Comment:
This probably has no effect, it returns the wrong token anyway (REJECT)

Reply:
Strange, as it works perfectly and it doesn't without it. 

I have no idea if option 90 in place of fc3118-auth works, I'll try it later.

Reply:
dhclient is weird oO

Reply:
option-90 does work. So if that be the case, then job is done. :)

All I did to test that was replace the "rfc3118-auth", "X"... in tables.c and recompile.

Makes one ask, why the hell put it in the original diff?

Reply:
Not enough time to understand dhclient I think. Like the BPF filter, too hard, XXX do later. But I can somewhat relate :)